metaflow/datatools/s3.py

```diff
@@ -309,6 +309,10 @@ def __init__(
 
         if not boto_found:
             raise MetaflowException("You need to install 'boto3' in order to use S3.")
+        
+        if tmproot is None:
+            artifact_localroot = from_conf("METAFLOW_ARTIFACT_LOCALROOT")
+            tmproot = artifact_localroot if artifact_localroot else "."
```

Comment:
If no PVC is mounted would this path still be valid as an ephemeral storage path?

Reply:
yes that is where the logs are put anyhow:

https://github.com/zillow/metaflow/blob/376ef787388fd3942986b2db25f819a3922b1ef9/metaflow/plugins/kfp/kfp_constants.py#L24